### PR TITLE
Cut release 0.3.0

### PR DESCRIPTION
**Pull request by beorn7**
@discordianfish @sdurrheimer

### CODE REVIEW

**Comment by discordianfish** [comment]
LGTM!

File: .gitignore [hunk 1]
```diff
@@ -1,9 +1,4 @@
 /pushgateway
-dependencies-stamp
-/.build
```
Comment (sdurrheimer) [inline .gitignore]: Please keep `/.build`, `/.release` and `/.tarballs`, they are used by the release process.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,15 @@
+## 0.3.0 / 2016-06-07
+* [CHANGE] Push now rejects improper and reserved labels.
+* [CHANGE] Required labels flag removed.
+* [BUGFIX] Docker image actually works now.
+* [ENHANCEMENT] Converted to Promu build process.
```
Comment (grobie) [inline CHANGELOG.md]: Could even be listed as change, as the structure of the release tarball changes.

Comment (beorn7) [inline CHANGELOG.md]: @grobie Please read line 6.

File: .gitignore (hunk 1 above)
Comment (beorn7) [inline .gitignore]: Oops, thanks. I thought those were leftovers.

**Comment by beorn7** [comment]
@sdurrheimer Addressed your comment. Does it look good otherwise?

File: .gitignore [hunk 3]
```diff
@@ -1,9 +1,7 @@
 /pushgateway
-dependencies-stamp
 /.build
-/.release
+/.release
```
Comment (sdurrheimer) [inline .gitignore]: whitespace ?

**Comment by sdurrheimer** [comment]
nits, LGTM otherwise.

File: .gitignore (hunk 3 above)
Comment (beorn7) [inline .gitignore]: Oops. How can emacs not fix that...

**Comment by beorn7** [comment]
thanks. will merge after fixing the whitespace thing.

File: CHANGELOG.md (hunk 2 above)
Comment (grobie) [inline CHANGELOG.md]: I thought I had deleted the comment. Sorry for the noise.

Comment (beorn7) [inline CHANGELOG.md]: No worries. It was even funny… ;)

**Pull request merged**